test/xpu_api/utilities/function.objects/arithmetic.operations/minus_arifm_op.pass.cpp

```diff
@@ -0,0 +1,56 @@
+// -*- C++ -*-
```

Comment:
Typo in the name of the file:
`minus_arifm_op.pass.cpp` -> `minus_arithm_op.pass.cpp`

Reply:
Fixed